packages/agents/sdk/src/sdkBase.ts

```diff
@@ -228,7 +228,17 @@ export class SdkBase extends SdkShared {
       });
     }
 
-    const connextContractAddress = (await this.getConnext(origin, options)).address;
+    const isValidAsset = await this.hasLockbox(origin, asset, options);
```

Comment:
let's name this something more specfic, like `isValidLockboxAsset`

Reply:
removed this whole logic